genai-perf/genai_perf/metrics/statistics.py

```diff
@@ -80,51 +80,50 @@ def _should_skip(self, data: List[Union[int, float]], attr: str) -> bool:
         return False
 
     def _calculate_mean(self, data: List[Union[int, float]], attr: str) -> None:
-        avg = np.mean(data)
+        avg = StatisticsUtil.calculate_mean(data)
         setattr(self, "avg_" + attr, avg)
-        self._stats_dict[attr]["avg"] = float(avg)
+        self._stats_dict[attr]["avg"] = avg
 
     def _calculate_percentiles(self, data: List[Union[int, float]], attr: str) -> None:
-        p25, p50, p75 = np.percentile(data, [25, 50, 75])
-        p90, p95, p99 = np.percentile(data, [90, 95, 99])
+        percentile_results = StatisticsUtil.calculate_percentiles(data)
```

Comment:
Do we want a wrapper around numpy like this?
Can you provide some insight into this portion of the design?